projects/kit/pipes/field-error/field-error-pipe.ts

```diff
@@ -132,17 +132,17 @@ export class TuiFieldErrorPipe implements PipeTransform, ControlValueAccessor {
             return of(new TuiValidationError(context));
         }
 
-        if (isObservable<PolymorpheusContent>(content)) {
-            return unwrapObservable(content, context);
+        if (isObservable(content)) {
+            return unwrapObservable(content as Observable<PolymorpheusContent>, context);
```

Comment:
new version isObservable returns `is Observable<unknown>`  https://github.com/ReactiveX/rxjs/blob/ad54bd807c35f13582194296e777208c3a71bd09/packages/rxjs/src/internal/Observable.ts#L1328

instead of `is Observable<T>` as before
https://github.com/ReactiveX/rxjs/blob/6.x/src/internal/util/isObservable.ts

Reply:
I think we can just change it to `instanceof Observable` then so we do not need to typecast. After all, that's all this function does in our case.